third_party/gems/known_gems.bzl

```diff
@@ -0,0 +1,71 @@
+KNOWN_GEM_SHA265 = {
```

Comment:
Can you add a comment here or in the other file that says this file is purely so that we don't print a bunch of stuff, but that even if something isn't listed in this file the tests will still work?